client/src/utilities/contactsimporter.cpp

```diff
@@ -31,6 +31,42 @@
 #include <QFile>
 #include "fatcrm_client_debug.h"
 #include <QTextCodec>
+#include <QLocale>
+#include <QStringList>
+#include <QString>
+
+#include "config-phonenumber.h"
+
+#if USE_PHONENUMBER
```

Comment:
Indenting inside #if is very strange and unusual ;)

IIRC an indented #include statement isn't even portable to all compilers.